cmd/root_test.go

```diff
@@ -59,7 +60,15 @@ func TestMain(m *testing.M) {
 		}
 	}()
 
-	// TODO: add https://github.com/uber-go/goleak
+	defer func() {
+		// TODO: figure out why logrus' `Entry.WriterLevel` goroutine sticks
+		// around and remove this exception.
+		opt := goleak.IgnoreTopFunction("io.(*pipe).read")
+		if err := goleak.Find(opt); err != nil {
+			fmt.Println(err) //nolint:forbidigo
+			exitCode = 3
+		}
+	}()
```

Comment:
I help here, I will take a look into it.

Reply:
For future reference, this will be resolved in https://github.com/grafana/k6/pull/2833, which I think we should keep as a separate PR, rather than trying to merge it in this series of other PRs?